### PR TITLE
Set params test

### PR DESCRIPTION
**Pull request by ahmad11111111111**
Hey. I think i fixed the issues, and it should be ok to merge. you can look over it and check.

### CODE REVIEW

File: pipeline/masked/psdbinner.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 from ..psdbinner import PSDBinner
 from typing import List, Tuple
+from sklearn.base import BaseEstimator, TransformerMixin  # type: ignore
 
-
-class MaskedPSDBinner(PSDBinner):
+class MaskedPSDBinner(PSDBinner,BaseEstimator):
```
Comment (kevincar) [inline pipeline/masked/psdbinner.py:7]: @ahmad11111111111 

Thanks for making this pull request. Is there a reason that you're reassigning the `BaseEstimator` class here? It is already assigned to the `PSDBinner`, which means by inheriting from `PSDBinner` the `MaskedPSDBinner` will also inherit from this class

**Comment by ahmad11111111111** [comment]
Hey I was trying to respond through outlook and the responses wouldn't work. And I just got the alert

**Comment by ahmad11111111111** [comment]
The reason I added that was that I added the base estimator to all of the masked classes. I didn't check the psd banners grandparent.

**Comment by kevincar** [comment]
@ahmad11111111111 

Good work! Two things:

1) See the comments I've added above, and address them where you can.
2) The checks are currently not passing. Can you do the following:
    1) run `python -m flake8 pipeline`. This will ensure that the code is formatted appropriately. You can actually [setup VSCode to auto format](https://marcobelo.medium.com/setting-up-python-black-on-visual-studio-code-5318eba4cd00#:~:text=Go%20to%20settings%20in%20your,”%20and%20select%20“black”.) each time you save a file 
    2) run `python -m mypy pipeline`. This will ensure that the code is type checked appropriately.

**Comment by ahmad11111111111** [comment]
trying to fix it now

**Comment by kevincar** [comment]
@ahmad11111111111 did you mean to close the pull request?

**Comment by ahmad11111111111** [comment]
i think i broke the merge with your additions. how do i revert it to your changes?

**Comment by ahmad11111111111** [comment]
no

**Comment by kevincar** [comment]
Well, let's just keep it simple. We'll keep this one closed and just keep the conversation going on #10
